### ISSUE TITLE
batch_norm should not be used in wgan_gp

### ISSUE THREAD

**Issue by wong-1994** [LixiangHan/GANs-for-1D-Signal#8]
batchnorm layer should not be used in wgan_gp
